src/Jackett.Common/Definitions/xbytes2.yml

```diff
@@ -57,6 +59,21 @@
             args: category
       title:
         selector: td[valign="middle"] a
+        filters:
+          - name: re_replace
+            args: ["S0(\\d{1,2})\\/E(\\d{1,2})", "$1x$2"]
+          - name: re_replace
+            args: ["\\/", " "]
+          - name: re_replace
+            args: ["\\(", ""]
+          - name: re_replace
+            args: ["\\)", ""]
+          - name: re_replace
+            args: ["2017", ""]
+          - name: re_replace
+            args: ["2018", ""]
+          - name: re_replace
+            args: ["2019", ""]
```

Comment:
I assume this is there in order to strip the year? sickrage can't handle years?
That sounds like a bad idea and will break other applications like sonarr/radarr.